src/registrar/admin.py

```diff
@@ -335,10 +343,11 @@ class UserDomainRoleAdmin(ListHeaderAdmin):
     search_fields = [
         "user__first_name",
         "user__last_name",
+        "user__email",
         "domain__name",
         "role",
```

Comment:
**question (non-blocking)**: We only have one role right now ("Manager") so I think that having that be a searchable field isn't getting us anything. But, it also isn't hurting anything so it probably isn't worth getting rid of it for now, and maybe when we grow multiple roles it will be useful.

Reply:
I thought the same thing @neilmb! Once we have more roles, I'd imagine that would become a filter (rather than a search). But no harm keeping it there, for now, I suppose.